packages/common-css/README.md

```diff
@@ -1,3 +1,37 @@
-# Fundamental Library Common CSS Package
+Fundamental Library Common CSS Package
 
-This package contains Common CSS files. More documentation to come soon...
+## What is Fundamental Library Common CSS?
+
+Fundamental Library Common CSS Package holds all the CSS style rules of the library that are recognized by older browsers—as well as the newer browsers. It creates the style rules for the basic elements on a page, such as a (link) elements, p (paragraph) elements, h1 (header) elements etc. The package is based on the DRY (Don't Repeat Yourself) methodology which allows the developers to not repeat the same CSS rules overall in the library.
+
```

Comment:
@InnaAtanasova Help us with description. Is this right?

Reply:
I don't think we're backwards compatible throughout the history of web browsers so I'd remove the bit about older and newer browsers. And I think if we're going to mention DRY let's also mention BEM

Reply:
Fundamental Library Common CSS Package is a set of utility classes and SCSS mixins that allow developers to add padding, margin, color, font, shadow, and more to their applications and controls without writing custom css. 
The class naming follows the BEM methodology and the values of the CSS rules are based on SAP Fiori Design Guidelines. Further customization is possible by using the SCSS mixins and providing user defined values as parameters.

Reply:
+1 on what Mike said

Reply:
@ynnenu what these two said 😁